README.md

```diff
@@ -40,7 +40,13 @@ $ chmod +x coveralls.phar
 To install php-coveralls with Composer, run the following command:
 
 ```sh
-$ composer require satooshi/php-coveralls --dev
+$ composer require satooshi/php-coveralls
```

Comment:
The current command wouldn't have worked in PHP 5.3. Hence the change to be in line with the other changes.